### PR TITLE
Fixes to body element animation

### PR DESCRIPTION
**Pull request by matsko**
[Review on Reviewable](https://reviewable.io/reviews/angular/angular.js/12245)

### CODE REVIEW

**Comment by lgalfaso** [comment]
I am slightly inclined into making the new `$body` service an undocumented service. That a side, LGTM

File: test/ngAnimate/animateSpec.js [hunk 1]
```diff
@@ -1209,6 +1209,34 @@ describe("animations", function() {
...
+        return function(element, method, options) {
+          capturedAnimation = arguments;
+          return new $$AnimateRunner();
+        };
+      });
+    });
+    inject(function($animate, $rootScope, $document, $rootElement) {
+      $animate.enabled(true);
+
+      var body = jqLite($document[0].body);
```
Comment (Narretz) [inline test/ngAnimate/animateSpec.js]: Does it make sense to use the new body service here instead?

**Comment by Narretz** [comment]
I also think $body should be $$body.

**Comment by matsko** [comment]
@petebacondarwin please review so we can merge this in tomorrow.

**Comment by petebacondarwin** [comment]
It looks fine to me. The only comment I have is that since `$$body` is only used once in the core `ng` module, we should move the definition of `$$body` over to the `ngAnimate` module so we don't need to make `angular.js` file any bigger.

**Comment by matsko** [comment]
Merged in as https://github.com/angular/angular.js/commit/44ce9c8288fc6c12043567027271a09bd0594d74. Excellent work @OlenDavis.

**Pull request closed without merging**